First Steps: Update Live Sample (#8335)
DevExpress/devextreme-documentation: concepts/Common/First Steps/25 Customize the App/00 Customize the App.md

> You can apply optional styles and templates to customize the app.

Reviewer: I think the word "optional" is redundant. I wouldn't think there are "mandatory" styles that I need to set up. 

And I would maybe clarify that you apply styles to DevExtreme components, and that's how you change the app appearance. At least that's what this document is about right? It applies styles to DevExtreme UI elements?

```suggestion
Apply styles and templates to DevExtreme components to customize your app appearance.
```

Author: Yes that is correct, the styles mentioned in "Customize the App" apply only to DevExtreme components.
